### PR TITLE
chatbox without api req is added

### PR DESCRIPTION
**Pull request by JoJosuk**

### CODE REVIEW

**Comment by JoJosuk** [comment]
#3

File: .vscode/settings.json
Comment (AshminJayson) [inline .vscode/settings.json:1]: Remove this from commit "git rm 'filename"

File: src/Components/Chatbox.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      sender: "user",
+      direction: "outgoing",
+    };
+
+    const messageList = [...messages, newmessage];
+    setMessages(messageList);
+    setTyping(true);
+  };
+
+  const gptimage = {
```
Comment (AshminJayson) [inline src/Components/Chatbox.tsx]: Move this outside function as it is static.

**Review by AshminJayson — CHANGES_REQUESTED** [review]

**Comment by JoJosuk** [comment]
changes resolved

**Review by AshminJayson — APPROVED** [review]

**Pull request merged**